botEngine.js

```diff
@@ -49,12 +49,13 @@ async function listenToMessages(client) {
      */
     let isAdminMessage = false;
     try {
-      isAdminMessage = message.member.roles.cache.some((r) => adminRoles.includes(r.name));
+      isAdminMessage = message.member.roles.cache
+        .some((r) => config.roles.adminRolesName.includes(r.name));
     } catch (e) {
       //  The only 'con' is a command or message gets ignored.
     }
 
-    const NOBOT_ROLE_ID = '783764176178774036';
+    const NOBOT_ROLE_ID = config.roles.NOBOTRoleId;
```

Comment:
Here and on L165 is the same weird declaring a constant that is only reused once pattern. I can see an argument for readability when it was just a string of number, but I think removing `NOBOT_ROLE_ID` in both instances and just calling `config.roles.NOBOTRoleId` for both uses should be fine

Reply:
See above comment ⏫